src/pages/overview/pdf-accessibility-auto-tag-api/quickstarts/dotnet/index.md

```diff
@@ -0,0 +1,333 @@
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <OutputType>Exe</OutputType>
+        <TargetFramework>netcoreapp3.1</TargetFramework>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="log4net" Version="2.0.12" />
+        <PackageReference Include="Adobe.PDFServicesSDK" Version="2.2.1" />
+    </ItemGroup>
+
+    <ItemGroup>
+        <None Update="pdfservices-api-credentials.json">
+            <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+        </None>
+        <None Update="private.key">
+            <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+        </None>
+        <None Update="extractPDFInput.pdf">
+            <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+        </None>
+        <None Update="log4net.config">
+            <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+        </None>
+    </ItemGroup>
+
+</Project>
+```
+
+Our application will take a PDF, `Adobe Extract API Sample.pdf` (downloadable from [here](/Adobe%20Extract%20API%20Sample.pdf)) and extract it's contents. The results will be saved as a ZIP file, `ExtractTextInfoFromPDF.zip`. We will then parse the results from the ZIP and print out the text of any `H1` headers found in the PDF.
+
+5) In your editor, open the directory where you previously copied the credentials and created the `csproj` file. Create a new file, `Program.cs`. 
+
+Now you're ready to begin coding.
+
+## Step Three: Creating the application
+
+1) We'll begin by including our required dependencies:
+
+```javascript
+using System.Text.Json;
+using System.IO.Compression;
+using System.IO;
+using System;
+using System.Collections.Generic;
+using log4net.Repository;
+using log4net.Config;
+using log4net;
+using System.Reflection;
+using Adobe.PDFServicesSDK;
+using Adobe.PDFServicesSDK.auth;
+using Adobe.PDFServicesSDK.pdfops;
+using Adobe.PDFServicesSDK.io;
+using Adobe.PDFServicesSDK.exception;
+using Adobe.PDFServicesSDK.options.extractpdf;
+```
+
+2) Now let's define our main class and `Main` method:
+
+```javascript
+namespace ExtractTextInfoFromPDF
+{
+    class Program
+    {
+        private static readonly ILog log = LogManager.GetLogger(typeof(Program));
+        static void Main()
+        {
+		}
+	}
+}
+```
+
+3) Now let's define our input and output:
+
+```javascript
+String input = "Adobe Extract API Sample.pdf";
+
+String output = "ExtractTextInfoFromPDF.zip";
+if(File.Exists(Directory.GetCurrentDirectory() + output))
+{
+	File.Delete(Directory.GetCurrentDirectory() + output);
+}
+```
+
+This defines what our output ZIP will be and optionally deletes it if it already exists. Then we define what PDF will be extracted. (You can download the source we used [here](/Adobe%20Extract%20API%20Sample.pdf).) In a real application, these values would be typically be dynamic. 

```

Comment:
Not going to add any more comments about this particular file as it looks like the wrong file was used.